includes/snmp.inc.php

```diff
@@ -134,6 +134,11 @@ function gen_snmpwalk_cmd($device, $oids, $options = null, $mib = null, $mibdir
         }
     }
 
+    $timeout = get_device_timeout($device);
```

Comment:
The old snmp.inc.php code already has a function called `prep_snmp_setting`, use that.

Reply:
Hmm, it looks like it should already be taking the device timeout into account. I'll check.

Reply:
Indeed it does. I'll remove the commit.